core/src/com/goonsquad/galactictd/screens/PlayScreen.java

```diff
@@ -24,9 +24,10 @@
 import com.goonsquad.galactictd.systems.positional.RotationSystem;
 
 public class PlayScreen implements Screen {
+    private static final float GAME_CAMERA_HEIGHT = 1080;
+    public static final float HUD_HEIGHT = 35f;
     public static final float GAME_WIDTH = 1920;
-    public static final float GAME_HEIGHT = 1080;
-    public static final float HUD_HEIGHT = 100f;
+    public static final float GAME_HEIGHT = 1080 - HUD_HEIGHT;
```

Comment:
These numbers don't actually change the size of screen, just the game world coordinates. The aspect ratio is import though.